com.jetbrains.PyCharm-Professional.appdata.xml

```diff
@@ -0,0 +1,156 @@
+					<li>Many fixes coming from WebStorm, DataGrip and IntelliJ IDEA</li>
+				</ul>
+			</description>
+		</release>
+		<release version="2018.3.1" date="2018-12-05">
+			<description>
+				<ul>
+					<li>A fix for the recently added WSL support in PyCharm 2018.3</li>
+					<li>An issue where PyCharm couldn’t correctly authenticate over SSH using a passphrase-protected private key has been resolved.</li>
+					<li>A few fixes for Docker and Docker Compose</li>
+					<li>Fixes for the embedded terminal</li>
+					<li>Many fixes coming from WebStorm, DataGrip and IntelliJ IDEA</li>
+				</ul>
+			</description>
+		</release>
+		<release version="2018.3" date="2018-11-21">
+			<description>
+				<ul>
+					<li>For those of you on Windows who are developing applications that run on Linux: Windows Subsystem for Linux is a quick and easy way to always have a Linux environment available. PyCharm 2018.3 can now be configured to use a Python interpreter inside WSL.</li>
+					<li>One of the most-requested features in our issue tracker were multiline TODOs, and we’re happy to announce that these are now available in PyCharm 2018.3.</li>
+					<li>Even though search everywhere (double-shift) is not at all a new feature, we’ve made some great usability improvements, and now it’s easy to see how you can narrow down the results.</li>
+				</ul>
+			</description>
+		</release>
+		<release version="2018.2.5" date="2018-11-14">
+			<description>
+				<ul>
+					<li>An issue that causes PyCharm to crash on Ubuntu 16.04 has been resolved</li>
+					<li>Matplotlib 3.0.0 can now be imported in the Python Console</li>
+					<li>Python code now folds correctly after it’s minimized with Ctrl+Shift+Numpad – (Cmd+Shift+- on macOS)</li>
+				</ul>
+			</description>
+		</release>
+		<release version="2018.2.4" date="2018-09-21">
+			<description>
+				<ul>
+					<li>Various small pipenv improvements</li>
+					<li>A bug in our pytest with fixtures support was fixed: previously, if yield statements were used in the fixture, PyCharm would assume that the return type of the function was a Generator. Now, the correct return type is inferred, preventing false positives.</li>
+				</ul>
+			</description>
+		</release>
+		<release version="2018.2.3" date="2018-09-06">
+			<description>
+				<ul>
+					<li>A number of improvements and fixes for the integrated Python console</li>
+					<li>Fixes for the new Quick Documentation window which was new in Pycharm 2018.2</li>
+					<li>Fixes for few false positives: install an already existing package intention, type checks</li>
+					<li>And much more, including improvements from the IntelliJ platform, WebStorm, and DataGrip, see the release notes for details.</li>
+				</ul>
+			</description>
+		</release>
+		<release version="2018.2.2" date="2018-08-21" />
+		<release version="2018.1.4" date="2018-06-10" />
+	</releases>
+	<project_license>Apache-2.0</project_license>
```

Comment:
Is this really actually Apache 2.0?

Reply:
Good catch.

I saw in [the guidelines](https://github.com/flathub/flathub/wiki/AppData-Guidelines) that the convention is to use `LicenseRef-proprietary`, but `LicenseRef` looks like a placeholder. Should I just put `proprietary` here?

Reply:
Nope, you should use `LicenseRef-proprietary`. I didn't make the spec ;-)